enclave/CMakeLists.txt

```diff
@@ -12,13 +12,18 @@ if (OE_SGX)
       ../common/sgx/report.c
       ../common/sgx/collateral.c
       ../common/sgx/sgxcertextensions.c
+      ../common/sgx/sgxmeasure.c
```

Comment:
I moved `host/sgx/sgxmeasure.c/.h` to `common/sgx/*`. During the verification of EEID reports in enclaves, we need to replay memory page additions, i.e. we need to be able to measure those instructions inside enclaves.